docs/migration/checklist.md

```diff
@@ -0,0 +1,26 @@
+# Migration Checklist
+
+### The Source Repo has 
+  - [ ] Dockerfile
+  - [ ] build_deploy.sh
+  - [ ] pr_check.sh
+  - [ ] e2e build disabled (after build pipeline is validated)
```

Comment:
I think e2e should be its own section titled "backwards compatibility"

Reply:
There's a few things to cover in e2e-deploy:
* You need to disable the 'quay copier' build config for your app (so that the app's image is not being pushed to quay from 2 places)
* You need to update your app's qa/smoke env YAML files so that your app pulls its image from quay -- if you completely get rid of all webhooks triggering the old BuildConfigs on OpenShift then you'd need to update the ci env YAML for this too.
* Disable the "prod" env option in your app's Jenkins deploy job

I can add more details for each of these to whatever doc we decide to house this info in ...